embedded_fab/content.dart

```diff
@@ -0,0 +1,415 @@
+    return Scaffold(
+      extendBody: true,
+      backgroundColor: Colors.white,
+      appBar: AppBar(
+        backgroundColor: Colors.white,
+        elevation: 0,
+        centerTitle: true,
+        title: Text(
+          widget.title,
+          style: textTheme.title.copyWith(
+            color: Colors.red,
+            fontWeight: FontWeight.w400,
+            fontSize: 24,
+          ),
+        ),
+        actions: <Widget>[
+          Padding(
+            padding: const EdgeInsets.all(12),
+            child: CircleAvatar(
+              backgroundColor: Colors.blueGrey.shade300,
+              radius: 16,
+              child: Text(
+                'AZ',
+                style: TextStyle(color: Colors.white),
+              ),
+            ),
+          )
+        ],
+      ),
+      bottomNavigationBar: BottomNavigationBar(
+        items: [
+          for (NavigationView view in _navigationViews)
+            BottomNavigationBarItem(
+              icon: view.icon,
+              title: Text(view.title),
+            ),
+        ],
+        currentIndex: _selectedIndex,
+        selectedItemColor: Colors.black,
+        unselectedItemColor: Colors.grey,
+        onTap: _onItemTapped,
+        type: BottomNavigationBarType.fixed,
+      ),
+      body: Center(
+        child: _navigationViews[_selectedIndex],
+      ),
+    );
+  }
+}
+
+class NavigationView extends StatelessWidget {
+  NavigationView({this.icon, this.title, this.order, this.times});
+
+  final Icon icon;
+  final String title;
+  final List<int> order;
+  final List<String> times;
+
+  final List<_ListItemContent> _itemContents = [
+    _ListItemContent(
+      title: 'Google Material Review',
+      name: 'Damien Correll',
+      description: 'Hey guys, here are all of the notes from today',
+      initials: 'DC',
+      color: Colors.red,
+      attachments: _ChipAttachments(),
+    ),
+    _ListItemContent(
+      title: 'Stickersheet sync',
+      name: 'Amy, David, Irin',
+      description: 'What\'s the latest version of the stickersheet',
+      initials: 'AR',
+      color: Colors.blue,
+    ),
+    _ListItemContent(
+      title: 'Architecture Article Photos',
+      name: 'David, Andrea, Sehee',
+      description: 'Hey all, as discussed here are the photos',
+      initials: 'DA',
+      color: Colors.purple,
+    ),
+    _ListItemContent(
+      title: 'Sketch Article',
+      name: 'Cortney Cassidy',
+      description: 'Found this on Twitter',
+      initials: 'CC',
+      color: Colors.green,
+    ),
+    _ListItemContent(
+      title: 'Stickersheet notes',
+      name: 'Andrea Bravo',
+      description: 'Were there any notes from the stickersheet?',
+      initials: 'AB',
+      color: Colors.pink,
+    ),
+    _ListItemContent(
+      title: 'New position',
+      name: 'Rachel Been',
+      description:
+          'I\'ve got exciting news! I\' been speaking with the Home team.',
```

Comment:
```suggestion
          "I've got exciting news! I've been speaking with the Home team.",
```

Consider using double quotes on just this line to avoid having to use escape characters. It's not _explicitly_ mentioned in the dart style guide, but there are examples in there that do just that.

https://dart.dev/guides/language/effective-dart/usage#avoid-using-curly-braces-in-interpolation-when-not-needed

Reply:
Done